lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.ts

```diff
@@ -214,6 +214,10 @@ export class CardViewTextItemComponent extends BaseCardView<CardViewTextItemMode
         return this.editable && this.property.editable;
     }
 
+    get notClickable(): boolean {
```

Comment:
you have `isClickable` below that has different logic in checks... why not using `!Clickable` or changing the name to something else